Dockerfile

```diff
@@ -11,3 +11,27 @@ RUN curl -sSf https://static.rust-lang.org/rustup.sh | sh
 ADD package_installs.R /tmp/package_installs.R
 RUN Rscript /tmp/package_installs.R && \
     bash -c "rm -Rf /tmp/Rtmp*"
+
+# Miniconda (used by time-series API)
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+RUN apt-get update --fix-missing && \
+    apt-get install -y wget bzip2 ca-certificates curl git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O ~/miniconda.sh && \
+    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda clean -tipsy && \
+    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+# Install packages (used by time-series API)
+RUN pip install pandas pycrypto
+
+ENV TINI_VERSION v0.16.1
```

Comment:
Is adding tini intentional here?  It doesn't seem to have to do with miniconda.  I don't know much about it, but the documentation seems like it could be a good idea anyway.